client/app/components/blocks/test-run-card/test-run-card.directive.js

```diff
@@ -442,9 +442,7 @@
                 }
 
                 function normalizePlatformData() {
-                    if (vm.testRun.config) {
-                        [vm.testRun.platformIcon, vm.testRun.platformVersion] = testsRunsService.refactorPlatformData(vm.testRun.config);
-                    }
+                    [vm.testRun.platformIcon, vm.testRun.platformVersion] = testsRunsService.refactorPlatformData(vm.testRun.config);
                 }
```

Comment:
This action should be removed from this directive like you have done it in `testsSessionCardController`

Reply:
fixed